_pages/robot/2018.html

```diff
@@ -1,44 +1,50 @@
+        </ul>
+        <li><strong>Climber:</strong> Our secondary objctive was to climb the scale. The following are the current mechanisms we are using to do so.</li>
+        <ul>
+          <!--TODO: Add the specific weight of the robot later.-->
+          <li><strong>Winch:</strong> Utilizing a massive gear reduction gearbox and a large hook, our robot has the capabillity to climb the scale.</li>
+          <!--TODO: Add more specific details about the hook and the tether.-->
+          <li><strong>Crane Hook:</strong> To fulfill our secondary objective--to climb the scale--we attached a large hook to the end of the robot arm. This hook is attached to a strap of webbing, which, in turn, is attached to </li>
+        </ul>
+        <!--TODO: get images or videos of the intake in action.-->
+        <li><strong>Power Cube Intake</strong> Our power cube intake is designed to have dual functionallity, both intaking and delivering power cubes to the portal. Currently, our intake is powered by two redline motors for the intake themselves, and two 72 rpm motors controlling the swinging movements of the intake arms themselves, with major changes expected in the near future.</li>
     </ul>
     <h5>Programming</h5>
-    <p>Just like in previous years, we used Python for our robot code. In addition to our robot code, we developed a dashboard for our touchscreen driver station. It is written in JavaScript/HTML/CSS, and communicates with the robot through a python-based tornado server. It is based on the <a href="https://github.com/FRCDashboard/FRCDashboard">FRCDashboard</a> framework we developed last year.</p>
-    <p>As an addendum to our dashboard, we used a company-donated <a href="https://vufine.com">VUFINE</a> wearable display to build a heads-up robot monitor which assisted our driver. The display allowed easy viewing of our robot's camera feed and some vital statistics of the robot. To build the display, we used a modified version of FRC Dashboard, connected with our standard dashboard through NetworkTables for ease of control.</p>
-    <p>We drive using a pilot-copilot operating system with one driver controlling robot motion and the other controlling all other robot functions. This allows each driver to control their own independent task, making operating the robot more efficient.</p>
-    <p>Our vision system uses newly introduced WPILib cscore to operate three cameras and preform image processing on gear vision target to deliver gears to the airship.</p>
+    <p>As in years past, our robot's code will be written in Python, while our driverstation (a graphical user interface that communicates between the drivers and the robot) utilizes a combination of JavaScript, HTML and CSS. The framework for the driverstation can be found <a href="https://github.com/FRCDashboard/FRCDashboard">FRCDashboard</a>here.</p>
```

Comment:
It's called a Dashboard, the driverstation is either a) the app FIRST provides to to help us communicate with our robot (this is the one where you can see things like the Enable button), or b) the literal touchscreen laptop.

Reply:
Also please use the same language we used last year about FRCDashboard; since our team created it we should plug it by name. (Change the "last year" bit though, of course.)

Reply:
Okay.